cloudformation/launchdetector.yaml

```diff
@@ -161,14 +175,16 @@ Resources:
         "Fn::Sub": |
           #!/bin/bash -e
 
-          apt-get -y update
-          apt-get -y install awscli openjdk-8-jre-headless
+          #this should be installed on the base box
```

Comment:
Use amigo to get this for free. Also commented lines can just be removed.

Reply:
yeah i have done, lines are left from before i put that in. will remove.